treescope/external/torch_support.py

```diff
@@ -74,12 +74,13 @@ def _truncate_and_copy(
       ignoring any axes whose slices are already computed in `source_slices`.
   """
   assert torch is not None, "PyTorch is not available."
+    
```

Comment:
Actually, sorry, could you also amend to remove this line with extra whitespace? It's tripping up the internal lint system and I haven't set up the external lint checks yet.

Reply:
Hopefully ok now!

Reply:
Thanks!